src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala

```diff
@@ -153,6 +154,35 @@ class RoutesPostgresRepoSpec extends FunSpec with BeforeAndAfter with Matchers w
             routes.map(_.id.get).toSet should be(Set(3))
           }
         }
+
+        describe("#selectDeletedBefore") {
+          it("should select nothing if there are no deleted routes") {
+            insertRoute("R1")
+            insertRoute("R2")
+
+            val dateTime: LocalDateTime = LocalDateTime.now().plusHours(1L)
+            val routes: List[RouteRow] = routesRepo.selectDeletedBefore(dateTime)
+
+            routes.size shouldBe 0
+          }
+
+          it("should select only routes that were deleted before the specified date") {
+            val insertedRoutes = Seq(insertRoute("R1"), insertRoute("R2"), insertRoute("R3"))
+
+            val now = LocalDateTime.now()
+            val lastDeletedAt = insertedRoutes.zipWithIndex
+              .map(routeRowWithIndex => {
+                val deletedAt = now.plusHours(routeRowWithIndex._2 + 1)
+                routeRowWithIndex._1.id.foreach(id => deleteRoute(id, Some(deletedAt)))
+                deletedAt
+              }).last
+
+            val routes: List[RouteRow] = routesRepo.selectDeletedBefore(lastDeletedAt)
+
+            routes.size shouldBe 2
+            routes.map(_.name) should contain theSameElementsAs Seq("R1", "R2")
```

Comment:
Another reason to use `should be` is that here you use `should contain` and not `shouldContain`